### DISCUSSION TITLE
umi4 使用@umijs/preset-vue qiankun插件报错

### DISCUSSION THREAD

**Discussion by MrSongXJ** [umijs/umi#10687] [Q&A]
在umi4的.umirc.ts文件中defineConfig设置presets: [require.resolve('@umijs/preset-vue')]后，通过以下方法开启qiankun
plugins: ['@umijs/plugins/dist/qiankun'],
qiankun: {
 master:{}
}

报错
export 'Navigate' (imported as 'Navigate') was not found in 'umi'
export 'useMatch' (imported as 'useMatch') was not found in 'umi'
查看代码后发现renderer-vue中没有export这两个方法

**Comment by fz6m** [comment]
现在 vue 不支持用 `Navigate` / `useMatch` 这些方法，这些是 react 的方法。

@xierenyuan 现在你们有 vue + qiankun 的使用实例么，能一起用不 🤔
